crates/task-impls/src/consensus/mod.rs

```diff
@@ -388,7 +370,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> ConsensusTaskState<TYPES, I>
                 if let Err(e) =
                     handle_quorum_proposal_validated(proposal, event_stream.clone(), self).await
                 {
-                    debug!("Failed to handle QuorumProposalValidated event {e:#}");
+                    warn!("Failed to handle QuorumProposalValidated event {e:#}");
```

Comment:
What made this upgrade in log level?

Reply:
From my reading of `handle_quorum_proposal_validated`, it doesn't introduce any errors except if `vote_if_able` fails. And I think we'd expect `vote_if_able` to fail at least once or twice per view depending on when it was called, so to me `warn!` feels like it would spam logs in deployment.

Reply:
`vote_if_able` is spawned in a seperate task so if it errors we will not return to this point and log.  I do think anything logged here would be something more unexpected.